src/service/rates.ts

```diff
@@ -105,7 +104,7 @@ export async function fiatPrices() {
 }
 
 async function fetchCELOPrice() {
-  return coinbase.getCELOPrice();
+  return await duel(getCoinMarketCapPrice("CELO"), coinbase.getDAIInUSD());
```

Comment:
we should get CELO price not DAI

Reply:
Thanks, this got introduced when I refactored.